mozjs/build/moz.configure/rust.configure

```diff
@@ -92,6 +92,8 @@ def rust_target(rust_compiler, rustc, target, cross_compiling):
             # Android
             ('x86', 'Android'): 'i686-linux-android',
             ('arm', 'Android'): 'arm-linux-androideabi',
+            ('aarch64', 'Android'): 'aarch64-linux-android',
+            ('armv7', 'Android'): 'arm-linux-androideabi',
```

Comment:
So the way to get those changes in this repo is first waiting for a merge in upstream and then running this? https://github.com/servo/mozjs#upgrading